R/navs-legacy.R

```diff
@@ -578,6 +590,7 @@ liTag <- function(id, title, value, icon) {
     tags$a(
       href = paste0("#", id),
       `data-toggle` = "tab",
+      `data-bs-toggle` = "tab",
```

Comment:
Add comment about which version of Bootstrap each attribute is for. (and same below)